kyuubi-server/web-ui/package.json

```diff
@@ -19,8 +19,10 @@
     "axios": "^0.27.2",
     "date-fns": "^2.29.3",
     "element-plus": "^2.2.12",
+    "monaco-editor": "^0.44.0",
```

Comment:
the ASF policy requires listing dips at LICENSE and NOTICE, for this case, please run `pnpm licenses list --prod` and update the content of `LICENSE-binary`

Reply:
ok

Reply:
Thanks for updating, the newly introduced deps are all under Catalog A, which looks great.

https://www.apache.org/legal/resolved.html#asf-3rd-party-license-policy